doc/zephyr.doxyfile.in

```diff
@@ -279,24 +279,22 @@ TAB_SIZE               = 8
 # with the commands \{ and \} for these it is advised to use the version @{ and
 # @} or use a double escape (\\{ and \\})
 
-ALIASES                = "rst=\verbatim embed:rst:leading-asterisk" \
-                         endrst=\endverbatim \
-                         "kconfig{1}=\htmlonly <code>\1</code> \endhtmlonly \xmlonly <verbatim>embed:rst:inline :kconfig:option:`\1`</verbatim> \endxmlonly" \
+ALIASES                = "kconfig{1}=\verbatim \1 \endverbatim" \
                          "req{1}=\ref ZEPH_\1 \"ZEPH-\1\"" \
                          "satisfy{1}=\xrefitem satisfy \"Satisfies requirement\" \"Requirement Implementation\" \1" \
                          "verify{1}=\xrefitem verify \"Verifies requirement\" \"Requirement Verification\" \1" \
                          "kconfig_dep{1}=\attention Available only when the following Kconfig option is enabled: \kconfig{\1}." \
                          "kconfig_dep{2}=\attention Available only when the following Kconfig options are enabled: \kconfig{\1}, \kconfig{\2}." \
                          "kconfig_dep{3}=\attention Available only when the following Kconfig options are enabled: \kconfig{\1}, \kconfig{\2}, \kconfig{\3}." \
                          "funcprops=\par \"Function properties (list may not be complete)\"" \
-                         "reschedule=\htmlonly reschedule \endhtmlonly \xmlonly <verbatim>embed:rst:inline :ref:`api_term_reschedule`</verbatim> \endxmlonly" \
-                         "sleep=\htmlonly sleep \endhtmlonly \xmlonly <verbatim>embed:rst:inline :ref:`api_term_sleep`</verbatim> \endxmlonly" \
-                         "no_wait=\htmlonly no-wait \endhtmlonly \xmlonly <verbatim>embed:rst:inline :ref:`api_term_no-wait`</verbatim> \endxmlonly" \
-                         "isr_ok=\htmlonly isr-ok \endhtmlonly \xmlonly <verbatim>embed:rst:inline :ref:`api_term_isr-ok`</verbatim> \endxmlonly" \
-                         "pre_kernel_ok=\htmlonly pre-kernel-ok \endhtmlonly \xmlonly <verbatim>embed:rst:inline :ref:`api_term_pre-kernel-ok`</verbatim> \endxmlonly" \
-                         "async=\htmlonly async \endhtmlonly \xmlonly <verbatim>embed:rst:inline :ref:`api_term_async`</verbatim> \endxmlonly" \
+                         "reschedule=\verbatim reschedule \endverbatim" \
+                         "sleep=\verbatim sleep \endverbatim" \
+                         "no_wait=\verbatim no-wait \endverbatim" \
+                         "isr_ok=\verbatim isr-ok \endverbatim" \
+                         "pre_kernel_ok=\verbatim pre-kernel-ok \endverbatim" \
+                         "async=\verbatim async \endverbatim" \
```

Comment:
```suggestion
                         "reschedule=\qualifier reschedule" \
                         "sleep=\qualifier sleep" \
                         "no_wait=\qualifier no-wait" \
                         "isr_ok=\qualifier isr-ok" \
                         "pre_kernel_ok=\qualifier pre-kernel-ok" \
                         "async=\qualifier async" \
```

Reply:
<img width="782" alt="image" src="https://github.com/user-attachments/assets/af89138d-0f2a-4924-96a3-0c4cad283867">

Reply:
nice, wasn't aware of this!

Reply:
> nice, wasn't aware of this!

ya it's so visually pleasing that it should almost give us all an incentive to *actually* be more systematic about using these :) Or actually maybe we should just ditch them since I'm not sure how helpful that is to users to really have only a small portion of the API annotated :/